### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by guest73** [arkon/ng-sidebar#97]
I did everything as described in the docs but when I start my app I get the following error in the console
```
zone.js:522 Unhandled Promise rejection: Can't resolve all parameters for Sidebar: ([object Object], [object Object], [object Object], ?). ; Zone: <root> ; Task: Promise.then ; Value: 
SyntaxError {__zone_symbol__error: Error: Can't resolve all parameters for Sidebar: ([object Object], [object Object], [object Object],……}
 Error: Can't resolve all parameters for Sidebar: ([object Object], [object Object], [object Object], ?).
```
My angular is of 2.4.10 version and the latest(5.1.0) version of ng-select.

**Comment by arkon** [comment]
Ah yes, it looks like I accidentally used something that was only introduced in Angular 4.x to check if it's running in a browser or not. You can either upgrade to Angular 4.x or wait until I can figure out a fix for that.

**Comment by arkon** [comment]
Fixed in v5.1.1.
